README.md

```diff
@@ -105,7 +105,10 @@ database:
   storage: '/home/myuser/.simple-breakpad-server/database.sqlite'
   logging: false
 customFields:
-  files: ['customfile']
+  files:
+    - 'customfile1'
```

Comment:
I think it's kind of confusing to document both ways of adding a custom field in the README. Let's keep one canonical way of adding a custom field and keep the old way for backwards compatibility only.